<ui version="4.0" >
 <class>SerialPanelClass</class>
 <widget class="QWidget" name="SerialPanelClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>339</width>
    <height>187</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>SerialPanel</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <widget class="QCheckBox" name="checkBox" >
     <property name="text" >
      <string>Usar porta serial?</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QWidget" native="1" name="widget" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_3" >
      <item row="0" column="0" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Porta :</string>
        </property>
        <property name="buddy" >
         <cstring>comPort</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>Veloc. Max.:</string>
        </property>
        <property name="buddy" >
         <cstring>velocMax</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string>B. Paridade :</string>
        </property>
        <property name="buddy" >
         <cstring>bParidade</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QComboBox" name="comPort" >
        <property name="editable" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QComboBox" name="velocMax" >
        <item>
         <property name="text" >
          <string>50</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>110</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>134</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>150</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>200</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>300</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>600</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>1200</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>1800</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>2400</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>4800</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>9600</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>14400</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>19200</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>38400</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>56000</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>57600</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>76800</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>115200</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>128000</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>256000</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="2" >
       <widget class="QComboBox" name="bParidade" >
        <item>
         <property name="text" >
          <string>Par</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Ímpar</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Nenhum</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Marca</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Espaço</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="label_4" >
        <property name="text" >
         <string>B.Dados :</string>
        </property>
        <property name="buddy" >
         <cstring>bDados</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QLabel" name="label_5" >
        <property name="text" >
         <string>B. Paragem :</string>
        </property>
        <property name="buddy" >
         <cstring>bParagem</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="2" >
       <widget class="QLabel" name="label_6" >
        <property name="text" >
         <string>Sequência p. gaveta :</string>
        </property>
        <property name="buddy" >
         <cstring>seqGaveta</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QComboBox" name="bDados" >
        <item>
         <property name="text" >
          <string>5</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>6</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>7</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>8</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="QComboBox" name="bParagem" >
        <item>
         <property name="text" >
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>1.5</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>2</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="2" >
       <widget class="QLineEdit" name="seqGaveta" />
      </item>
      <item row="4" column="1" >
       <widget class="QPushButton" name="pushButton" >
        <property name="text" >
         <string>Testar</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" >
       <widget class="QPushButton" name="pushButton_2" >
        <property name="text" >
         <string>Salvar</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2" >
       <widget class="QPushButton" name="pushButton_3" >
        <property name="text" >
         <string>Abrir Gaveta</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1" >
    <spacer name="horizontalSpacer" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>18</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" >
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>15</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>SerialPanelClass</receiver>
   <slot>save()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>61</x>
     <y>138</y>
    </hint>
    <hint type="destinationlabel" >
     <x>65</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>SerialPanelClass</receiver>
   <slot>test()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>138</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel" >
     <x>140</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>SerialPanelClass</receiver>
   <slot>abrirGaveta()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>241</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel" >
     <x>241</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>widget</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>70</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel" >
     <x>70</x>
     <y>35</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>save()</slot>
  <slot>test()</slot>
  <slot>abrirGaveta()</slot>
 </slots>
</ui>
